lib/jsdom/living/nodes/HTMLInputElement-impl.js

```diff
@@ -316,6 +320,16 @@ class HTMLInputElementImpl extends HTMLElementImpl {
     return this._otherRadioGroupElements.some(radioGroupElement => radioGroupElement.checked);
   }
 
+  get labels() {
+    const root = this.ownerDocument;
+    return NodeList.create([], {
+      element: root,
+      query: () => domSymbolTree.treeToArray(root, {
+        filter: node => node.tagName === "LABEL" && node.control === this
+      })
+    });
```

Comment:
Please always implement things according to the [spec](https://html.spec.whatwg.org/multipage/forms.html#dom-lfe-labels), which specifically says that the same object must be returned on separate invocations of `labels` getter, and that the `labels` getter should return null in case the type of this `input` element is Hidden.